pages/windows/netstat.md

```diff
@@ -0,0 +1,12 @@
+# netstat
+
+> Displays active TCP connections, ports on which the computer is listening, Ethernet statistics, the IP routing table, IPv4 statistics and IPv6 statistics.
+> More information: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/netstat>.
+
+- Displays active TCP connections:
+
+`netstat`
+
+- Displays a list of currently open ports and related IP addresses:
```

Comment:
Is this talking about listening ports? or all ports?

Reply:
It tells you what state the port is in: listening, established or closed

Reply:
```suggestion
- Display a list of currently open ports and related IP addresses:
```